### ISSUE TITLE
Getting channels from channel selection

### ISSUE THREAD

**Issue by ilonadem** [pyRiemann/pyRiemann#117]
I am using pyriemann channel selection in order to consider a smaller number of channels to use later on in my code. 

**Is it possible to get a list of the selected channels, or channel indices that contribute to the reduced covariance matrix after channel selection, instead of just the reduced covariance matrix?**

This is what my code looks like:

```
cov1 = pyriemann.estimation.Covariances('oas').fit_transform(data1)
y1 = np.zeros((np.shape(cov1)[0]))
for i in range(len(y1)): y1[i] = 1

cov2 = pyriemann.estimation.Covariances('oas').fit_transform(data2)
y2 = np.zeros((np.shape(cov2)[0]))
for i in range(len(y2)): y2[i] = 2

Cov = np.concatenate((cov1, cov2), axis=0)
y = np.concatenate((y1, y2), axis=0)

selector = pyriemann.channelselection.ElectrodeSelection(20)
selector.fit(Cov, y)
Cov_new = selector.transform(Cov)
params_to_keep = selector.get_params()
```

When I print `params_to_keep`, I get:
`{'metric': 'riemann', 'n_jobs': 1, 'nelec': 20}`

**Comment by ilonadem** [comment]
Nevermind! I just adapted the `pyriemann.channelselection.ElectrodeSelection` class to return the indices of the kept channels alongside the covariance matrix.

**Comment by agramfort** [comment]
fit is allowed to set instance attributes that end with an underscore _

feel free to change the code and send us a PR

>
